doc/topics/releases/templates/3007.0.md.template

```diff
@@ -12,9 +12,6 @@ In Salt 3009, the ``setup.py`` file will be stripped of it's custom additions an
 or whatever is found best during the process of removing the customizations. <br>
 **If you're relying on these customizations please stop as your workflow will break in the future**.
 
-## Python 3.11
```

Comment:
It probably should be given that Salt is two major releases behind. 3.11 brings some huge performance improvements

Reply:
> 3.11 brings some huge performance improvements

Sadly, with regards to Salt, we saw exactly the opposite.